main.py

```diff
@@ -2,6 +2,30 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres_to_create = ["Western", "Action", "Dramma"]
+    for genre in genres_to_create:
+        Genre.objects.create(name=genre)
+
+    actors_to_create = [
+        ("George", "Klooney"), ("Kianu", "Reaves"), ("Scarlett", "Keegan"),
```

Comment:
would make sense to place each tuple on new line here since you have so many of them